app/frontend/src/features/messages/surfing/HostRequestListItem.tsx

```diff
@@ -32,9 +37,7 @@ export default function HostRequestListItem({
         {otherUser && <Avatar user={otherUser} />}
         <Box className={classes.card}>
           <Box className={classes.root}>
-            {otherUser && (
-              <UserName user={otherUser} className={classes.name} />
-            )}
+            {otherUser && <Typography className={classes.name}></Typography>}
```

Comment:
```suggestion
            {otherUser && <Typography className={classes.name}>{otherUser.name}</Typography>}
```
Probably still want to display the other user's name here?

Reply:
Or is this not the case since you deleted the `UserName` component?

Reply:
I just left it for the next PR, it's done there